### ISSUE TITLE
Review skipped tests

### ISSUE THREAD

**Issue by fozy81** [BiologicalRecordsCentre/NBN4R#4]
Some tests are skipped or skipped on cran - check these are not breaking and review if they should be skipped or not.

**Comment by AugustT** [comment]
Tests that depend on an external server (lots here I would think) are usually skipped on CRAN to avoid CRAN getting angry with us when the server goes down.

**Comment by AugustT** [comment]
We can set up a nightly build on Travis that does test these so we get notifications straight away if there are issues.

**Comment by raymondben** [comment]
Yup, tests that actually make calls to the servers are skipped on CRAN (in ALA4R, and hence are probably skipped here too). Tests are also skipped on CRAN if they take more than a few seconds, or download significant amounts of data (but all of these tests run on Travis and Appveyor). And for the same reason, lots of examples are wrapped in `\dontrun{...}` so that CRAN doesn't run them.
